keygen/src/keygen.rs

```diff
@@ -664,6 +694,15 @@ fn do_main(matches: &ArgMatches<'_>) -> Result<(), Box<dyn error::Error>> {
 
             let use_mnemonic = matches.is_present("use_mnemonic");
 
+            let (use_derivation_path, derivation_path) =
+                match matches.value_of("use_derivation_path") {
+                    Some(derivation_path) => (
+                        true,
+                        Some(DerivationPath::from_absolute_path_str(derivation_path).unwrap()),
+                    ),
+                    None => (false, None),
```

Comment:
I think this path will never be hit because `.default_value("m/44/501/0/0")` is provided in the clap.rs setup, which doesn't seem to be the intent for the rest of this patch

Reply:
I'm sorry, but I didn't understand (((
Do you think nobody will use grind with some value in --derivation-path other than default? I'm sorry (((

Reply:
Ohh, I'm a lamer 🤣 
Thank you for explanation, I fully understand where I was stupid)))